Press.php

```diff
@@ -0,0 +1,121 @@
+							<ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+								<li><a class="dropdown-item" href="aboutus.php">About Us</a></li>
+								<li><a class="dropdown-item" href="faq.php">FAQ</a></li>
+								<li><a class="dropdown-item" href="contact.php">Contact Us</a></li>
+								<li><a class="dropdown-item" href="merch.php">Merchandise</a></li>
+							</ul>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+	</header>
+    <div class="container">
+        <div class="row my-4">
+            <a href="https://stepoutbuffalo.com/bar-bill-as-good-as-it-gets-chicken-wings-buffalo/" target="_blank">
+                <div class="card bg-dark text-white p-0 ">
+                    <img src="images/press_step_out.jpg" class="card-img headerimg" alt="stepout">
+                    <div class="card-img-overlay">
+                        <h2 class="card-title">Stepout Buffalo</h2>
+                        <p class="card-text p-5 bg-custom rounded">"I’m going to put this simply. Bar Bill wings are as good as they get."</p>
+                    </div>
+                </div>
+            </a>
+        </div>
+        <div class="row my-4">
+            <a href="https://stepoutbuffalo.com/bar-bill-as-good-as-it-gets-chicken-wings-buffalo/" target="_blank">
+                <div class="card bg-dark text-white">
+                    <img src="images/press_roadfood.jpg" class="card-img headerimg" alt="Roadfood">
+                    <div class="card-img-overlay">
+                        <h2 class="card-title">RoadFood</h2>
+                        <p class="card-text p-5 bg-custom rounded">"On the endlessly appetizing Buffalo Wing Trail, this place is a highlight."</p>
+                    </div>
+                </div>
+            </a>
+        </div>
+        <div class="row my-4">
+            <a href="https://stepoutbuffalo.com/bar-bill-as-good-as-it-gets-chicken-wings-buffalo/" target="_blank">
+                <div class="card bg-dark text-dark">
+                    <img src="images/press_thrillist.jfif" class="card-img headerimg" alt="Thrillist">
+                    <div class="card-img-overlay">
+                        <h2 class="card-title">Thrillist</h2>
+                        <p class="card-text text-white p-5 bg-custom rounded"> "New York State is the birthplace of Buffalo wings... All roads lead to Anchor Bar in Buffalo, New York"</p>
+                    </div>
+                </div>
+            </a>
+        </div>
+    </div>
+    <footer class="container-fluid">
+		<div class="footer">
+			<h2>Our Locations</h2>
+			<p><em>East Aurora Takeout</em><br />
+				191 Main Street, Rear<br />
+				East Aurora, NY 14052<br />
+				(716) 652-7959
+			</p>
+			<p><em>East Aurora</em><br />
+				185 Main Street<br />
+				East Aurora, NY 14052<br />
+				(716) 652-7959
+			</p>
+			<p><em>Clarence</em><br />
+				8326 Main Street<br />
+				Clarence, NY 14221<br />
+				(716) 710-9464
+			</p>
+		</div>
+	</footer>
```

Comment:
also seeing that this is on more than one page, and is fairly static, this should get wrapped into a seperate file.